package.json

```diff
@@ -22,12 +22,12 @@
 		"enum": "^2.5.0",
 		"express": "^4.15.3",
 		"express-rate-limit": "^5.0.0",
```

Comment:
This is a just comment.
You don't need change these version
https://www.npmjs.com/package/semver